reference/events.v1.yaml

```diff
@@ -86,32 +129,54 @@ components:
       description: |-
         An event is a notification that something of interest has happened within a service, that, based on authorised scopes, a subscriber may then process.  This enables systems to communicate with each other in a loosely coupled way, where the event creator has no direct connection to any of the subscribing systems.
 
-        Typically an event will contain a basic data object that indicates what the state of the object is at the point at which the event is raised, but this is optional.  In the case that no data object exists it is intended that the subscribing system would call back to the provided API to retrieve the information.
+        In the SEM context the events either send information on an event that happenened (usage, progress, results) or the latest status of an object that is passed for reference (SIS, Product related  - Student, teacher, group, products)
```

Comment:
In the last category should the Courses also be included?

Reply:
we never defined events for courses.... Should we ?

Reply:
Digital content can be updated during the schoolyear - either because of customer feedback or because publishers werent finished in time and are adding additional content later on.
So it feels like we need those events - probably a notification that something changed for a certain course, and then it is up to the receiver to request the CMI5.
And when thinking about customer specific learning paths, then it maybe triggered by users on the learning platforms (where the learning path is created by the teacher and then pushed to the LMS where the students can launch it)?

Reply:
Ok, I will add it